autograder-core/src/test/java/de/firemage/autograder/core/check/structure/TestTooFewPackagesCheck.java

```diff
@@ -0,0 +1,83 @@
+package de.firemage.autograder.core.check.structure;
+
+import de.firemage.autograder.core.LinterException;
+import de.firemage.autograder.core.LocalizedMessage;
+import de.firemage.autograder.core.Problem;
+import de.firemage.autograder.core.ProblemType;
+import de.firemage.autograder.core.check.AbstractCheckTest;
+import de.firemage.autograder.core.compiler.JavaVersion;
+import de.firemage.autograder.core.file.StringSourceInfo;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TestTooFewPackagesCheck extends AbstractCheckTest {
+    private static final String LOCALIZED_MESSAGE_KEY = "too-few-packages";
+    @Test
+    void test() throws IOException, LinterException {
+        List<Problem> problems = super.check(StringSourceInfo.fromSourceStrings(
```

Comment:
```suggestion
        ProblemIterator problems = this.checkIterator(StringSourceInfo.fromSourceStrings(
```
`super` makes IntelliJ complain. I know that some checks do this, haven't taken the time to fix them and make them use the checkIterator method instead.